tst/functions/assertions/HaveParameter.Tests.ps1

```diff
@@ -305,6 +307,38 @@ InPesterModuleScope {
                 $err.Exception.Message | Verify-Equal "Expected command Invoke-DummyFunction to have a parameter ParamWithNotNullOrEmptyValidation, which is mandatory, of type [System.TimeSpan], the default value to be 'wrong value' and has ArgumentCompletion, because of reasons, but it wasn't mandatory, it was of type [System.DateTime], the default value was '(Get-Date)' and has no ArgumentCompletion."
             }
         }
+
+        Context 'Using InParameterSet' {
```

Comment:
We should add a test for `InParameterSet` on a function that does not have any parameter set at all (for example a function that does not have any parameters at all). To validate that it does not throw any exceptions.

Reply:
See other comments. You have to specify a parameter to use this operator at all. 🙂